WORKSPACE

```diff
@@ -24,27 +25,39 @@ load("@rules_jvm_external//:defs.bzl", "maven_install")
 maven_install(
     artifacts = [
         "org.jetbrains.kotlin:kotlin-stdlib:{}".format(_KOTLIN_COMPILER_VERSION),
-        "androidx.core:core-ktx:1.3.2",
-        "androidx.appcompat:appcompat:1.2.0",
-        "com.google.android.material:material:1.2.1",
-        "androidx.activity:activity-compose:1.3.0-alpha02",
-        "androidx.compose.material:material:1.0.0-beta08",
-        "androidx.compose.ui:ui:1.0.0-beta08",
-        "androidx.compose.ui:ui-tooling:1.0.0-beta08",
-        "androidx.compose.compiler:compiler:1.0.0-beta08",
-        "androidx.compose.runtime:runtime:1.0.0-beta08",
+        "androidx.core:core-ktx:1.6.0",
+        "androidx.appcompat:appcompat:1.3.1",
+        "com.google.android.material:material:1.4.0",
+        "androidx.activity:activity-compose:1.3.0",
+        "androidx.compose.material:material:{}".format(_COMPOSE_VERSION),
+        "androidx.compose.ui:ui:{}".format(_COMPOSE_VERSION),
+        "androidx.compose.ui:ui-tooling:{}".format(_COMPOSE_VERSION),
+        "androidx.compose.compiler:compiler:{}".format(_COMPOSE_VERSION),
+        "androidx.compose.runtime:runtime:{}".format(_COMPOSE_VERSION),
     ],
     override_targets = {
         "org.jetbrains.kotlin:kotlin-stdlib": "@com_github_jetbrains_kotlin//:kotlin-stdlib",
         "org.jetbrains.kotlin:kotlin-stdlib-jdk7": "@com_github_jetbrains_kotlin//:kotlin-stdlib-jdk7",
         "org.jetbrains.kotlin:kotlin-stdlib-jdk8": "@com_github_jetbrains_kotlin//:kotlin-stdlib-jdk8",
```

Comment:
I wonder, what was the purpose of adding overrides for kotlin-stdlib artifacts?
The project can be built without them as well

Reply:
You can remove the overrides

Reply:
Removed the overrides